react-native-iot-wifi.podspec

```
@@ -0,0 +1,18 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '../package.json')))
```

since the file is now in the `root directory` the reference for the package.json needs to be at the same level as the podspec file.
```suggestion
package = JSON.parse(File.read(File.join(__dir__, './package.json')))
```